.gitignore

```diff
@@ -462,3 +462,6 @@ dask-worker-space/
 *.pub
 *.rdp
 *_rsa
+
+# swig jni
+*_swig.jnilib
```

Comment:
Interesting, were you using SWIG builds to test your changes?

Anyway, can you please open a separate PR proposing this `.gitignore` change, and explaining how files like this get created and why they shouldn't be checked into the repo?

We have a strong preference in this project for small, focused pull requests, and this change doesn't seem tightly related to the core purpose of this PR.

Reply:
removed it from the change set.

Reply:
i think this ended up here bc of the way i built it locally. 70/30 chance it was due to me not having looked/thought about c++ for 10 years. ignore the .gitignore